ledger/accountdb.go

```diff
@@ -1366,6 +1376,11 @@ func accountsCreateCatchpointFirstStageInfoTable(ctx context.Context, e db.Execu
 	return err
 }
 
+func accountsCreateStateProofVerificationTable(ctx context.Context, e db.Executable) error {
```

Comment:
Not sure we need the word accounts at the begining

Reply:
All creation functions begin with accounts. Would you like me to deviate from the convention? Could make looking up my function harder.